setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='django-bulstyle',
-    version='1.3a1',
+    version='1.4a1',
```

Comment:
I would suggest changing the version to 1.3a2. We haven't released 1.3 yet - these can be prereleases for 1.3 until we're ready to release 1.3 final.